src/common/config_opts.h

```diff
@@ -408,6 +408,8 @@ OPTION(mon_client_max_log_entries_per_message, OPT_INT, 1000)
 OPTION(mon_max_pool_pg_num, OPT_INT, 65536)
 OPTION(mon_pool_quota_warn_threshold, OPT_INT, 0) // percent of quota at which to issue warnings
 OPTION(mon_pool_quota_crit_threshold, OPT_INT, 0) // percent of quota at which to issue errors
+OPTION(mon_pool_warn_size, OPT_INT, 1) // pool size below which to issue warnings
+OPTION(mon_pool_warn_min_size, OPT_INT, 1) // pool min size below which to issue warnings
```

Comment:
nit: teh comment shoudl say "pool size at which to issue warnings" (not "below which", since the check is <= not <)